### PR TITLE
Update app_languages.php

### PR DESCRIPTION
**Pull request by JoinFreeMe**
Filled in all language fields with the appropriate translation.

### CODE REVIEW

File: app/system/app_languages.php [hunk 1]
```diff
@@ -91,74 +91,74 @@
...
-$text['label-path']['uk'] = "";
-$text['label-path']['de-at'] = "";
+$text['label-path']['es-cl'] = "Ruta del proyecto";
+$text['label-path']['pt-pt'] = "Caminho do projeto";
+$text['label-path']['fr-fr'] = "Chemin du projet";
+$text['label-path']['pt-br'] = "Caminho do projeto";
+$text['label-path']['pl'] = "Ścieżka projektu";
+$text['label-path']['sv-se'] = "projekt Path";
+$text['label-path']['uk'] = "шлях проекту";
+$text['label-path']['de-at'] = "шлях проекту";
```
Comment (moteus) [inline app/system/app_languages.php]: I think this is wrong

**Comment by markjcrane** [comment]
I think this is right. Translators can come in for each language and improve on them at a later date.

**Comment by moteus** [comment]
I mean only like `$text['label-path']['de-at'] = "шлях проекту"` I belive this is not German :)

**Comment by markjcrane** [comment]
Ah well you are right on that for sure.

**Comment by JoinFreeMe** [comment]
Commit was made to correct this

**Comment by JoinFreeMe** [comment]
And uk is Ukraine I believe as informed by mcrane

**Comment by moteus** [comment]
Uk is united kindom. Ua is ukraine

**Comment by JoinFreeMe** [comment]
In which case I'll commit again when I get back to a machine

**Comment by reliberate** [comment]
Does Google have it wrong then?  See URL...   https://translate.google.com/m/translate#auto/uk/Happy

**Comment by moteus** [comment]
Seems you are right. https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes.

**Comment by markjcrane** [comment]
I see we are in agreement. This is also an interesting link on this topic ISO standard for UA as well as alternative ISO standard for UK. United Kingdom is actually GB.
http://www.loc.gov/standards/iso639-2/php/code_list.php

**Comment by JoinFreeMe** [comment]
Thank you, I've bookmarked this for future reference :)

**Pull request merged**